package.json

```diff
@@ -2,8 +2,10 @@
   "name": "application-name",
   "version": "0.0.1",
   "dependencies": {
+    "azure": "^1.2.0-preview",
```

Comment:
And you thought: thats just fine? Please make sure you use something stable. We don't want to debug later.